### ISSUE TITLE
Tabula question

### ISSUE THREAD

**Issue by jupton66** [tabulapdf/tabula#737]
Hi,

I'm trying to unlock a 380-ish page PDF, and I'm noticing records that break the line are entirely dropped on Tabula's import. As a result, I'm missing several hundred records. 

For example, 
<img width="762" alt="screen shot 2017-10-11 at 6 33 09 pm" src="https://user-images.githubusercontent.com/26164759/31470622-e5740830-aeb2-11e7-9c13-fc45383ba9a3.png">

(I'm not sure if you can see the photo attached, but it shows the red Tabula capture crossing a page break). 

But in the export, the record that was sourced from USA TODAY (11 June 2016 , Rif Dimashq, "USA TODAY...) is completely missing. 
<img width="763" alt="screen shot 2017-10-11 at 6 32 14 pm" src="https://user-images.githubusercontent.com/26164759/31470713-313d4ccc-aeb3-11e7-961d-cee4870dfdf4.png">

Any work arounds for this? 

Thanks!

Jodi

**Comment by jupton66** [comment]
FWIW, I have tried getting the red Tabula capture boxes to butt up against each other. It makes no difference.

**Comment by jeremybmerrill** [comment]
Yep, that's a known issue, unfortunately. The fact that there's no ruling line on the bottom of the cell (on that page) means Tabula's Lattice mode won't recognize it as a cell at all.

The only workaround I can suggest is using Stream mode to extract the data in those rows. It'll be a pain in the butt to "roll up" all of the text that belongs to the same row (since Stream mode will treat each line of text as its own row), but it'll actually extract the data. It's not a great workaround, I know.

(We've talked _somewhere_ about "pretending" that the user's selection is bounding lines for this purpose... but that might introduce other bugs and complexities. I remember talking about it but can't find the discussion to link to this one.)

This is a similar issue to #578. (Which I'm noting mostly for my own organization purposes.)
